### DISCUSSION TITLE
Long names in relationship mapping are not easily accessible

### DISCUSSION THREAD

**Discussion by kropsi** [directus/directus#7118] [Feature Requests]
### Problem

Long names of the tables are not visible and not easily accessible to check for any errors or a proper naming

![CleanShot 2021-08-02 at 10 41 41](https://user-images.githubusercontent.com/946662/127832201-4f1d9b9a-6098-4eda-8af7-4d1694f07626.png)

The problem is that if the name is not correct then you need to delete the field and collection and start over.

### Possible solution
It could be easily fixed with a title attribute, so the names could appear when hovering over the inputfield.

**Comment by benhaynes** [comment]
I agree — we should add a tooltip to these inputs so you can always see the full value. 👍

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
